scripts/generate.py

```diff
@@ -0,0 +1,58 @@
+"""
+Used for generating the repository from scratch.
+"""
+from pathlib import Path
+
+# from cookiecutter.main import cookiecutter
+from prefect_collection_generator.gql import populate_collection_repo
+
+THIS_DIRECTORY = Path(__file__).parent.absolute()
+REPO_DIRECTORY = THIS_DIRECTORY.parent
+
+# USE THIS IF NEED TO REGENERATE FROM SCRATCH; IF NOT SKIP TO NEXT SECTION
+# extra_context = {
+#     "full_name":  "Prefect Technologies, Inc.",
+#     "email": "help@prefect.io",
+#     "github_organization": "PrefectHQ",
+#     "collection_name": "prefect-github",
+#     "collection_short_description": "Prefect integrations interacting with GitHub",  # noqa
+# }
+
+# collection_template_url = "https://github.com/PrefectHQ/prefect-collection-template"
+# cookiecutter(
+#     collection_template_url,
+#     no_input=True,
+#     checkout="generated_graphql",
+#     extra_context=extra_context,
+#     overwrite_if_exists=True
+# )
+# REPO_DIRECTORY = THIS_DIRECTORY / "prefect_github"  # redirects repo_directory
+
+# UPDATE THESE AS DESIRED
+
+service_name = "GitHub"  # e.g. GitHub
+service_url = "https://api.github.com/graphql"  # e.g. https://api.github.com/graphql
```

Comment:
Probably don't need these comments
```suggestion
service_name = "GitHub"
service_url = "https://api.github.com/graphql"
```